doc-content/enterprise-only/openshift/proc-dashbuilder-standalone.adoc

```diff
@@ -0,0 +1,77 @@
+[id="proc-dashbuilder-standalone_{context}"]
+= Installing Dashbuilder Standalone using Operator
+
+You can use Operator to install Dashbuilder Standalone independently from other services.
+
+.Procedure
+
+. On the *Installation* page, in the *Application name* field, enter `standalone-dashbuilder`.
+. In the *Environment* field, enter `rhpam-standalone-dashbuilder`.
+. Click *Next*.
+. On the *Security* page, click *Next*.
+. On the *Components* page, from the *Components* list, select *Dashbuilder*.
+. To add a {KIE_SERVER} dataset, perform the following tasks:
+.. Click *Add new {KIE_SERVER} DataSets*.
+.. In the *DataSet name* field, enter `kieserver-1`.
+.. In the *Kie Server Location* field, enter the location of your {KIE_SERVER}, for example `https://my-kie-server:80/services/rest/server`.
+.. To set your credentials, perform one of the following tasks:
+* If you do not have a token set, in the *Username* and *Password* fields, enter your username and password. Leave the *Token* field blank.
+* If you have a token, in the *Token* field, enter your token. Leave the *Username* and *Password* fields blank.
++
+The custom resource example:
++
+----
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: standalone-dashbuilder
+spec:
+  environment: rhpam-standalone-dashbuilder
+  objects:
+    dashbuilder:
+      config:
+        kieServerDataSets:
+          - name: kieserver-1
+            location: 'https://my-kie-server:80/services/rest/server'
+            user: kieserverAdmin
+            password: kieserverAdminPwd
+            replaceQuery: true
+----
+
++
+NOTE: You can add multiple {KIE_SERVER} DataSets by repeating step 6.
```

Comment:
For this instance and the next one (step 7) you should never hard-code a step number. Create an anchor to the step and then provide the proper link syntax. Unfortunately, I do not know how to do it, but I did find this online:

How to Link to a Specific Part of a Page
Give the object or text you'd like to link to a name. ...
Take the name you've chosen and insert it into an opening HTML anchor link tag. ...
Place that complete opening <a> tag from above before the text or object you want to link to, and add a closing </a> tag after.

Reply:
Agreed, changing this by removing hard-code reference. I couldn't get the link syntax to work unfortunately.